crates/op-rbuilder/src/payload_builder_vanilla.rs

```diff
@@ -76,15 +76,20 @@ use std::{fmt::Display, sync::Arc, time::Instant};
 use tokio_util::sync::CancellationToken;
 use tracing::{info, trace, warn};
 
-#[derive(Debug, Clone, Copy, Default)]
+#[derive(Debug, Clone, Default)]
 #[non_exhaustive]
 pub struct CustomOpPayloadBuilder {
     builder_signer: Option<Signer>,
+    #[allow(dead_code)]
```

Comment:
I think you can have this field enabled only if the flashblocks feature is enabled?